.terraform-version

```diff
@@ -1 +1 @@
-0.14.5
\ No newline at end of file
+1.0.0
```

Comment:
this should be symlinked to the component folder so that if you run it locally it gets picked up